src/data/nat/factorial.lean

```diff
@@ -123,4 +123,24 @@ begin
   { exact add_factorial_lt_factorial_add _ hi }
 end
 
+lemma add_factorial_le_factorial_add_succ (i : ℕ) (n : ℕ) :
```

Comment:
Maybe a bit wordy, but:
```suggestion
lemma add_factorial_succ_le_factorial_add_succ (i : ℕ) (n : ℕ) :
```
or if that's too long
```suggestion
lemma add_factorial_succ_le (i : ℕ) (n : ℕ) :
```
Maybe leave it to another reviewer to decide.

Reply:
To be honest, I am happy with either choice.  The lemma has a place in the Liouville PR, but I am not so sure about wider applicability.